packages/remix-dev/index.ts

```diff
@@ -1,6 +1,6 @@
 import "./modules";
 
-export type { AppConfig } from "./config";
+export type { AppConfig, RemixConfig as ResolvedRemixConfig } from "./config";
```

Comment:
We probably don't want users importing `RemixConfig` but it'll be useful for us internally as features expand. The `ResolvedRemixConfig` alias clarifies intent and makes it less likely for users to confuse it with `AppConfig`.